macros/macros.in

```diff
@@ -234,14 +234,14 @@ package or when debugging this package.\
 %_exec_prefix		%{_prefix}
 %_bindir		%{_exec_prefix}/bin
 %_sbindir		%{_exec_prefix}/sbin
-%_libexecdir		%{_exec_prefix}/libexec
+%_libexecdir		%{_exec_prefix}/lib
 %_datadir		%{_prefix}/share
 %_docdir		%{_datadir}/doc
 %_sysconfdir		/etc
 %_sharedstatedir	%{_prefix}/com
 %_localstatedir		/var
 %_lib			lib
-%_libdir		%{_exec_prefix}/%{_lib}
+%_libdir		%{_exec_prefix}/%{_lib}/%({__dpkg_architecture} -qDEB_HOST_MULTIARCH)
```

Comment:
I missed a `%`. Fixed that.

That said, %{_libexecdir} is defined as `%{_prefix}/lib` on Debian systems, which this should now indicate. So if you need things to go into `/usr/lib/foo`, you should use `%_libexecdir`. Debian FHS indicates that things should *not* be in `/usr/lib` itself, either in `/usr/lib/<name>` (for libexec stuff) or `/usr/lib/<triple>` for libraries.

In the past, this invocation didn't work like it was supposed to. Now that it does, we can use this to make sure it gets the right information at build-time.